scripts/install_r.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+echo -e "
+Welcome to the Containerised R Development Environment
+
+Here you can modify, compile and use R as described in
+the r development guide:
+    https://contributor.r-project.org/rdevguide/GetStart.html#building-r
+
+To help you, we have already set the following environment variables:
+
+BUILDDIR = \"$BUILDDIR\"
+TOP_SRCDIR = \"$TOP_SRCDIR\"
+
+Have fun \U0001F601
+"
+
+echo "Do you want to install trunk version of R?(Y/N)"
+read answer
+
+if [ "$answer" = "Y" ]; then
+    echo "Installing the latest R trunk version."
+    svn checkout https://svn.r-project.org/R/trunk/ "$TOP_SRCDIR"
+    mkdir -p "$BUILDDIR"
+    cd "$BUILDDIR"
+
+    "$TOP_SRCDIR/configure" --enable-R-shlib --without-recommended-packages
+
+    make check
+    make
+
+    #changing terminal path
+
+# Path to the settings.json file
+    settings_file_path="/home/vscode/.vscode-remote/data/Machine/settings.json"
+
+ 
+# Read the existing JSON content
+    settings_data=$(cat "$settings_file_path")
+
+
+# Update the desired setting using jq (a command-line JSON processor)
+    updated_settings_data=$(echo "$settings_data" | jq '."r.rterm.linux"="/usr/bin/R"')
+
+ 
+# Write the updated JSON content back to the file
+   echo "$updated_settings_data" > "$settings_file_path"
+    
+else
+    echo "If you want to install R trunk version please type Y by re-running install_r.sh script !"
+fi
+
+
+echo "Installed R-devel trunk"
```

Comment:
Can you take this section out for the moment (save the code in a separate script for possible inclusion in a later PR). The behaviour on first starting the dev container should be just to print the welcome message. Then users should work through the README to build R-devel for the first time, as in v1.0 of the container.

We might provide the commands to build R in a script for convenience. But I think this should be used by people that are already familiar with building R and I don't think this script should be interactive (i.e. controlled by questions and answers). Instead, the script would be there as a reference, that a user could run all at once by sourcing the script, or they could customise, e.g. with their own choice of configuration settings, before running. In any case, that workflow is a separate issue from what we addressing in this PR, so we shouldn't change too much at once.